src/strings/csv/en.csv

```diff
@@ -447,6 +447,7 @@ GERMINATING_QUANTITY,Germinating Quantity,
 GERMINATING_QUANTITY_REQUIRED,Germinating Quantity *,
 GERMINATION_QUANTITY_AFTER,Germination Quantity (After),
 GERMINATION_QUANTITY_BEFORE,Germination Quantity (Before),
+GERMINATING_QUANTITY_MUST_BE_GREATER_THAN_ZERO,Germinating quantity must be greater than 0
```

Comment:
thoughts on 'cannot be less than 0'? it can be equal to 0 (can be greater than or equal to 0)... same for the other case

Reply:
Good catch thanks!